jwst/datamodels/schemas/firstframe.schema.yaml

```diff
@@ -0,0 +1,26 @@
+title: First Frame Correction data model
```

Comment:
Because the first-frame step doesn't use a reference file there's no need to define a reference file schema. This new file (firstframe.schema.yaml) can be removed.

Reply:
Same answer/question as above.